config/samples/dataplane_v1beta1_openstackdataplanerole.yaml

```diff
@@ -4,58 +4,8 @@ metadata:
   name: openstackdataplanerole-sample
 spec:
   services:
-    - configure-network
-    - validate-network
     - install-os
-    - configure-os
-    - run-os
   nodeTemplate:
-    networkConfig:
-      template: |
```

Comment:
why removing this?

Reply:
Bad rebase most likely. I'll put it back.